### DISCUSSION TITLE
What is the issue in implementing the folder/file view?

### DISCUSSION THREAD

**Discussion by qkhanhpro** [jellyfin/jellyfin#6771] [Q&A]
I see it as the single top requested feature https://features.jellyfin.org/posts/224/add-a-folder-view-to-libraries
with comment refering to it as "low hanging fruit" but there seems to be no try to implement it yet?

Is there any specific difficulty in approaching this? Can we discuss it here?
If there is no obvioius blocking point in this I think I will try to implement it

**Comment by dkanada** [comment]
@qkhanhpro the comment about "low hanging fruit" was probably not made by someone with experience in this codebase, but I can tell you that it *is* possible without any server changes. The pull request would only involve a toggle to enable the existing folder view for every library rather than only including it for unsupported libraries such as books.

It's not a "true" folder view but it would be the low hanging fruit you describe since it's barely different than using the `Other` library type when creating a new library.

**Reply by cvium** [reply]
The existing folder view is really bad and if I'm honest it doesn't look like Emby has made any noticeable improvements to theirs either.

**Reply by qkhanhpro** [reply]
Thanks @dkanada , @cvium 
Can you point me to the flag on the web project?
Maybe I can see how broken it is and try to improve upon it?

**Comment by cvium** [comment]
Whoever is calling it "low hanging fruit" should just go ahead and show us the way :) Keep in mind that we do not accept database changes to `library.db` such as, but not limited to, new columns and new tables.
